devices/pipe-rtt/device.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * RTT pipes: communication through debug probe (device)
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <devices/devs.h>
+#include <hal/hal.h>
+#include <lib/lib.h>
+
+
+extern int rtt_check(int chan);
+extern ssize_t rtt_write(int chan, const void *buf, size_t count);
+extern ssize_t rtt_read(int chan, void *buf, size_t count);
```

Comment:
Why not provide these prototypes in a header file?

Reply:
Done.